helpers/loginUser.ts

```diff
@@ -1,15 +1,20 @@
 const SERVER_URL = process.env.SERVER_URL
 
 export const loginUser = async (username: string, password: string) => {
-  const res = await fetch(`${SERVER_URL}/signin`, {
+  const res = await fetch(`${SERVER_URL}`, {
```

Comment:
Should we be using the `react/apollo` hooks `useMutation` and `useQuery` for the new backend?